modal/mount.py

```diff
@@ -670,19 +691,19 @@ def _get_client_mount():
     )
 }
 
+
 SYS_PREFIXES |= {p.resolve() for p in SYS_PREFIXES}
 
+MODAL_PACKAGES = ["modal", "modal_proto", "modal_version"]
```

Comment:
We should fix this so that the client distribution contains only one package btw, would be more compatible with PEP517.

Reply:
Yeah for sure